prov/shm/src/smr_ep.c

```diff
@@ -478,7 +478,7 @@ static int smr_progress_cmd_rma(struct smr_ep *ep, struct smr_cmd *cmd)
 	    ofi_cirque_isfull(ep->util_ep.rx_cq->cirq)) {
 		FI_WARN(&smr_prov, FI_LOG_EP_CTRL,
 			"rx cq full\n");
-		ret = -FI_ENOSPC;
+		return -FI_ENOSPC;
```

Comment:
Is this return value ever reported back to the user?

Reply:
never mind - I just looked.  If things are working correctly, we should never hit this anyway -- at least I think

Reply:
Yeah, these error codes are only used internally to indicate whether to continue to progress or not and this one I don't think should ever be an issue. I can take it out later if I determine this for sure.
Also, any idea why AppVeyor is failing?

Reply:
No idea -- it stopped on 1 of the 6 builds for some reason without a useful error message.  I restarted them.